pages/issuing/cardholder_terms.en.md

```diff
@@ -0,0 +1,176 @@
+
+### Authorized Card Users
+
+You are responsible for all authorized transactions initiated and fees incurred by use of your Card. If you permit another person to have access to your Card or Card number, we will treat this as if you have authorized such use and you will be liable for all transactions and fees incurred by those persons. You are wholly responsible for the use of each Card according to the terms and conditions of this Agreement.
+
+### Secondary Cardholder
+
+You may not request an additional Card for another person. Each family member must have their own Card Account.
+
+### Your Representations and Warranties
+
+By activating the Card or by retaining, using or authorizing the use of the Card, you represent and warrant to us that: 
+
+ 1. you are at least 15 years of age;
+ 2. you are a legal resident residing in Cambodia;
+ 3. you have provided us with a verifiable Photo ID Card, a Cambodia street address;
+ 4. the personal information that you provide to us in connection with the Card is true, correct and complete;
+ 5. you received a copy of this Agreement and agree to be bound by and to comply     with its terms; and complete;
+ 6. you accept the Card.
+
+
+A minor less than 15 years of age or without a government issued photo ID; a legal guardian must submit a parental sponsorship application before the card is activated. 
+
+### Cash Access
+
+With your PIN, you may use your Card to obtain cash from any Automated Teller Machine ("ATM") or any Point-of-Sale ("POS") device that supports cash-out, as permissible by a merchant, that bears the UnionPay<sup>®</sup> or BONGLOY<sup>®</sup> Acceptance Mark. All ATM transactions are treated as cash withdrawal transactions.  
+
+| Type of card                                  | Prepaid Card | Virtual        |
+|:---------------------------------------------:|:------------:|:--------------:|
+| Cash withdrawal limit (per day)               | US$1000      | Not Applicable |
+| Purchase limit-Off line (per day)             | US$2,500     | US$2,500       |
+| Purchase limit-On line (per day)              | US$500       | US$500         |
+| Maximum cash withdrawal transaction (per day) | 2 times      | Not Applicable |
+
+### Loading Your Card
+
+You may add funds to your Card, called "value loading", at any time. The minimum amount of the initial value load is US$5.00. The Card must be loaded for at least the minimum amount at the time the Card is purchased at the branch location. The minimum amount of each value reload is US$5.00 via agent partner or online via the consumer website. The maximum amount of the initial value load at the time of Card purchase is US$500.00 at the agent partner location. The maximum amount of each value reload is US$2,500.00. Online via the consumer website the maximum reload amount is US$500.00 per transaction; up to US$2,500.00 per day.
```

Comment:
@chandapen this doesn't make sense for us, we can't support topup online. We should remove the parts which suggest they can topup online.